frontend/main.js

```diff
@@ -175,6 +205,13 @@ async function startApp () {
     }
   })
 
+  sbp('chelonia/externalStateSetup', { reactiveSet: Vue.set, reactiveDel: Vue.delete })
```

Comment:
Shouldn't this be done before we setup the web socket via `chelonia/configure`?

Reply:
Not necessarily, because there's no state to sync yet. But it could be done.

Reply:
I would move it there just to be safe